starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,47 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:700|Roboto:100,300,700');
+
+#first{
+    background-image: url("../images/apple-pie.jpg"); 
+    background-size: cover;
+    width: 25cm;
+    height: 15cm;
+    align-content: center;
+    margin: 20px 250px 10px 250px;
+    justify-content: center;
+}
+    
+    h1 {text-align:center; font-family:Roboto;
+    font-size:50pt; 
+    color:white;
+    padding:240px;
+ }
+
+
+.second{text-align: center; 
+    font-family: Roboto ;
+    font-size:15pt;
+    padding: 20px 350px 0px 350px;
+    text-align: justify}
+
+#image1{text-align:center}
```

Comment:
because `#image1` and `#image2` have the same css properties, you could turn this into a class